docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx

```diff
@@ -0,0 +1,62 @@
+
+## Using the Ethereum integration
+
+To interact with the Ethereum network or another EVM-compatible network, a developer will need to use a combination of the management canister API and EVM RPC canister RPC methods to complete the following steps:
+
+1. [Generate a public key](generating-addresses.mdx): To sign and submit transactions, your canister will need a public key. You will also need this public key to generate an Ethereum wallet address.
+
+2. [Sign a transaction](signing-transactions.mdx): Next, a transaction must be signed before it can be submitted to the Ethereum network and executed. Transactions are signed using a canister's public key.
+
+3. [Submit a transaction](submit-transactions.mdx): Signed raw transactions can be submitted using the EVM RPC canister's RPC method `eth_sendRawTransaction`. Code examples for sending transactions can be found in the [EVM RPC canister](/docs/current/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister) documentation.
+
+4. [Read information from the Ethereum network](/docs/current/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister): Information from the Ethereum network can be queried using RPC methods such as:
+
+    - `eth_getBlockByNumber`: Returns information about a given block.
+
+    - `eth_getTransactionCount`: Returns the number of transactions for a specified address.
+
+    - `eth_getTransactionReceipt`: Returns details about a submitted transaction.
+
+    - `eth_feeHistory`: Returns the historical fee data to estimate gas prices for transactions.
+
+    - `eth_getLog`s: Returns the logs of a specified block or transaction.
+
+Each of these steps can be executed programmatically within a smart contract that is deployed on ICP. View the [EVM RPC canister](/docs/current/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister) documentation to see code examples for each RPC method.
+
+## Integrating with Ethereum
+
+The Ethereum integration can be incorporated into dapps for several purposes, such as authentication, as shown in the [Sign In With Ethereum](siwe.mdx) example. Additional sample projects can be found in the [EVM RPC](../evm-rpc/samples.mdx) canister documentation.
+
+## Using ckETH
+
+Chain-key Ethereum (ckETH) is a digital twin token of Ethereum that is backed 1:1 with Ethereum held within an ICP smart contract. ckETH tokens comply with the [ICRC token standard](/docs/current/developer-docs/multi-chain/ethereum/cketh/overview).
```

Comment:
Should we also mention ckUSDCSepolia as well?

Reply:
Added info about ckUSDCSepolia